### ISSUE TITLE
Prod Issue - Can't display

### ISSUE THREAD

**Issue by Code4SocialGood** [Code4SocialGood/c4sg-web#1179]
**Scenario**
Go to prod site: https://app.code4socialgood.org/
Click Organizations
Click 35Kcampaign
Click user name in Contact section in the sidebar
Page can't display. Error in console.
(we loaded the prod data to dev, but this issue can't be recreated in dev. must be some data is different between dev and prod)

**Analysis**
This error happens when the user's profile hasn't finished yet.
1. When a new org user is logged in: if you click Create Project, it displayed: Please complete your user profile and organization profile before posting projects
2. Click My Organization and complete organization profile
3. Click Create Project: it didn't display any warning under project image, although the user profile has not been created
4. If use fill in the page on Create Project, user could have the project created, which should be prevented since user profile is not completed yet.

Please only allow project to be created when both user profile and org profile are completed. If any of them are not completed, please display warning message under image on the Create Project page. 

Please also only allow organization to be created after user profile is complete. Otherwise, display warning on the My Organization page.

In summary, here is the sequence that user must follow before he can move to the next step:
1. Finish user profile
2. Finish organization profile
3. Create Project

![image](https://user-images.githubusercontent.com/24813620/29945558-2fb9eebc-8e67-11e7-9af9-df8822a01827.png)

![image](https://user-images.githubusercontent.com/24813620/29945604-659847cc-8e67-11e7-9205-5eb7c1a285d1.png)

![image](https://user-images.githubusercontent.com/24813620/29854814-c2e25d7e-8d0d-11e7-8c25-483edb8b27eb.png)

![image](https://user-images.githubusercontent.com/24813620/29854868-06fa6998-8d0e-11e7-8e32-c775e82a57eb.png)

![image](https://user-images.githubusercontent.com/24813620/29854881-1623db0c-8d0e-11e7-84a3-5116a1d0534a.png)

**Comment by Code4SocialGood** [comment]
It happens for this organization user as well:

![image](https://user-images.githubusercontent.com/24813620/29944877-b07d9c90-8e64-11e7-80d3-849a044a8459.png)

**Comment by Code4SocialGood** [comment]
@Kiranpreet The PR
